packages/hub/src/users.spec.ts

```diff
@@ -131,17 +138,18 @@ describe('Users...', () => {
         expect(err).to.not.equal(wrongError)
         expect(err.code).to.equal(grpc.Code.Unauthenticated)
       }
-      // No key signature
+      /**
+       * No key signature will pass because default security is null
+       */
       const tmp = new Context(addrApiurl).withSession(dev.session)
       const key = await createKey(tmp, 'ACCOUNT')
       try {
         await client.listThreads(ctx.withAPIKey(key.key))
         throw wrongError
       } catch (err) {
-        expect(err).to.not.equal(wrongError)
-        expect(err.code).to.equal(grpc.Code.Unauthenticated)
```

Comment:
same here, should this be unauthenticated sometime in the future?

Reply:
yep. hopefully will update